engine/lite/api/src/main/scala/pl/touk/nussknacker/engine/lite/api/utils/sources.scala

```diff
@@ -0,0 +1,39 @@
+package pl.touk.nussknacker.engine.lite.api.utils
+
+import cats.Monad
+import cats.data.{Validated, ValidatedNel}
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.NodeId
+import pl.touk.nussknacker.engine.api.exception.NuExceptionInfo
+import pl.touk.nussknacker.engine.api.runtimecontext.{ContextIdGenerator, EngineRuntimeContext}
+import pl.touk.nussknacker.engine.api.{Context, Lifecycle}
+import pl.touk.nussknacker.engine.lite.api.commonTypes.ErrorType
+import pl.touk.nussknacker.engine.lite.api.customComponentTypes
+import pl.touk.nussknacker.engine.lite.api.customComponentTypes.LiteSource
+
+import scala.language.higherKinds
+import scala.util.Try
+
+object sources {
```

Comment:
Can we use just package and one file in it? Until there is no need to have some package scope scala "type"s, those objects introduce a lot of problems e.g. you can't have two the same objects in the same packages when you split sth between two modules - we have such problems during moving classes between flink and non-flink modules

Reply:
:+1: done